### DISCUSSION TITLE
Showing touch indicator in multiple charts simultaneously

### DISCUSSION THREAD

**Discussion by kubaolszewski** [imaNNeo/fl_chart#1413] [Q&A] [answered]
I want to create a ListView with several charts displaying different sets of data, the main case is to make them work so that If I click on one of the charts the touch indicator is shown on all currently visible charts at the same spot on X axis. Can't get through it, I wonder if it is actually available with this package, will appreciate any kind of help or advice. I also added the video with the effect I want to achieve created in different flutter package for charts display. 

https://github.com/imaNNeo/fl_chart/assets/118610589/50a93550-bcb6-4c46-b6ba-fb9e2525c7dc

@imaNNeo

**Comment by imaNNeo** [comment, marked as answer]
You can customize the touch callback.
[Check the source-code here](https://gist.github.com/imaNNeo/946acd2c1718bec0c42e5cdede598868)

https://github.com/imaNNeo/fl_chart/assets/7009300/43546fad-5c09-409f-bb8a-5635c1df336d

**Comment by kubaolszewski** [comment]
thank you so much, I have another question in this situation - what about option of zooming on chart to see bigger chunks of chart with implementation of previous functionalities? is it somehow possible right now? @imaNNeo

**Reply by imaNNeo** [reply]
For zooming, you can follow #71
